tests/menus/context_menu_comonent.test.ts

```diff
@@ -594,17 +604,74 @@ describe("Context Menu internal tests", () => {
     ]);
     await renderContextMenu(300, 300, { menuItems });
 
-    triggerMouseEvent(".o-menu div[data-name='root_1']", "mouseover");
-    await nextTick();
-    triggerMouseEvent(".o-menu div[data-name='root_1_1']", "mouseover");
-    await nextTick();
+    await mouseOverMenuElement(".o-menu div[data-name='root_1']");
+    await mouseOverMenuElement(".o-menu div[data-name='root_1_1']");
     expect(fixture.querySelector(".o-menu div[data-name='subMenu_1']")).toBeTruthy();
-    triggerMouseEvent(".o-menu div[data-name='root_2']", "mouseover");
-    await nextTick();
+    await mouseOverMenuElement(".o-menu div[data-name='root_2']");
     expect(fixture.querySelector(".o-menu div[data-name='subMenu_1']")).toBeFalsy();
     expect(fixture.querySelector(".o-menu div[data-name='root_2_1']")).toBeTruthy();
   });
 
+  test("Submenus don't close immediately", async () => {
+    const menuItems = createActions([
```

Comment:
Any way to take less than 50 lines to define the menuItems ? 😅 

children can be an array, it doesn't have to be a function that returns an array. Probably worth to create an helper `makeTestMenu(name: string, params: Maybe<ActionSpec>) ` or something like that. If you're motivated you can also shorten the other menu items declaration of the test file. We can probably easily save some 100 lines of code